app/js/enhancedDate.js

```diff
@@ -1,5 +1,52 @@
-/* 
- * add your module definition for enhancedDate here
- * assign the module to a global variable
- * /
+var enhancedDate = (function() {    
+    var myDate = new Date();    
+    myDate.setHours(0, 0, 0, 0);
+
+    return {    
+        getDate: function() {   
+            return myDate.toDateString();
```

Comment:
This function can return the date in one of two formats: either as a millisecond value, or a Date object. If the user passes "true" to the getDate function, it should return the Date as an object, otherwise it should return the millisecond value.